modules/UI/UI.js

```diff
@@ -298,6 +298,8 @@ UI.getSharedVideoManager = function () {
  * esablished, false - otherwise (for example in the case of welcome page)
  */
 UI.start = function () {
+    UI.stopDaemons();
```

Comment:
It's not really needed other than for safety. Like before you create an interval or timeout on an instance, you first clear any that exist already. I'll remove it.